### ISSUE TITLE
Unable to get evolu web working with SolidStart

### ISSUE THREAD

**Issue by ibb2** [evoluhq/evolu#444]
Hi again, I have been trying to get Evolu implemented in my Solid js application, however have been unsucessful. So I have created a minimal test repo. With Evolu installed and the example from the docs implemented I keep getting the following error:
```
[plugin:solid] /Users/ibrahim/Developer/Test/solid-project-evolu/node_modules/.vinxi/client/deps/chunk-VKMYUPOW.js: 

Unexpected token, expected "=>" (11799:10)
  11802 | });
```

I have added `@evolu/common-web` to vite.config.ts however I am still unable to get the project to run. 

Test repo
https://github.com/ibb2/solid-project-evolu

**Comment by steida** [comment]
Thank you for reporting that. Did you try to take a look at the Evolu example Vite config? Or what has been changed recently? I'm in the middle of implementing much faster and more scalable sync, with zero time to play with Vite configuration.

**Comment by ibb2** [comment]
Yeah I've looked at the vite example, the code looks similar to mine so I'm not sure what Im doing wrong. And I'm not sure what you mean by 

> Or what has been changed recently

I started the project maybe a week ago and made the test repo around when I made this issue, so there hasn't been any releases since then.
